src/components/TimeAccountingApplet.jsx

```diff
@@ -0,0 +1,294 @@
+  exposuresLoading,
+  almanacLoading,
+  sumExpTime,
+  nightHours,
+  exposureFields,
+  weatherLoss,
+  faultLoss,
+}) {
+  const expPercent = Math.round((sumExpTime / (nightHours * 3600)) * 100);
+  const nonExpPercent = 100 - expPercent;
+
+  const data = useMemo(
+    () =>
+      // fallback to empty array if exposureFields is undefined
+      (exposureFields ?? []).map((entry) => {
+        // Convert obs_start and obs_end to DateTime objects
+        const obsStart = entry["obs_start"];
+        const obsEnd = entry["obs_end"];
+        let obs_start_dt = undefined;
+        let obs_end_dt = undefined;
+        if (
+          typeof obsStart === "string" &&
+          DateTime.fromISO(obsStart).isValid
+        ) {
+          obs_start_dt = DateTime.fromISO(obsStart);
+        }
+
+        if (typeof obsEnd === "string" && DateTime.fromISO(obsEnd).isValid) {
+          obs_end_dt = DateTime.fromISO(obsEnd);
+        }
+        return { ...entry, obs_start_dt, obs_end_dt };
+      }),
+    [exposureFields],
+  );
+
+  const sortedData = useMemo(
+    () =>
+      // fallback to empty array if data is undefined
+      (data ?? [])
+        .filter((d) => d.obs_start_dt.isValid && d.obs_end_dt.isValid)
+        .sort((a, b) => a.obs_start_dt - b.obs_start_dt),
+    [data],
+  );
+
+  const [gapWithFilterChange, gapWithoutFilterChange] = useMemo(() => {
+    let filterChange = 0;
+    let noFilterChange = 0;
+    for (let i = 0; i < sortedData.length - 1; i++) {
+      const currentExp = sortedData[i];
+      const nextExp = sortedData[i + 1];
+      if (nextExp.day_obs !== currentExp.day_obs) continue; // skip if not the same night
+      const gap = nextExp.obs_start_dt - currentExp.obs_end_dt;
+      if (currentExp.band === nextExp.band) {
+        noFilterChange += gap;
+      } else {
+        filterChange += gap;
+      }
+    }
+    return [filterChange / 3600000, noFilterChange / 3600000]; // convert ms to hours
+  }, [sortedData]);
+
+  const chartConfig = {
+    gaps: {
+      label: "Inter-Exposure time (same filter)",
+      color: "hsl(200, 70%, 50%)",
+    },
+    gaps_filter_change: {
+      label: "Inter-Exposure time (with filter change)",
+      color: "hsl(40, 70%, 50%)",
+    },
+    fault: { label: "Fault", color: "hsl(0, 70%, 50%)" },
+    weather: { label: "Weather", color: "hsl(80, 70%, 50%)" },
+  };
+  const chartData = [
+    {
+      name: "Gaps",
+      value: gapWithoutFilterChange,
+      color: "hsl(200, 70%, 50%)",
+      label: "Inter-Exposure time (same filter)",
+    },
+    {
+      name: "Gaps (Filter)",
+      value: gapWithFilterChange,
+      color: "hsl(40, 70%, 50%)",
+      label: "Inter-Exposure time (with filter change)",
+    },
+    {
+      name: "Fault",
+      value: faultLoss,
+      color: "hsl(0, 70%, 50%)",
+      label: "Time Lost to Faults",
+    },
+    {
+      name: "Weather",
+      value: weatherLoss,
+      color: "hsl(80, 70%, 50%)",
+      label: "Time lost to Weather",
```

Comment:
```suggestion
      label: "Time Lost to Weather",
```
Or the above -> `Time lost to Faults`